### DISCUSSION TITLE
Debug output in output pane

### DISCUSSION THREAD

**Discussion by matthewjcavalier** [forcedotcom/salesforcedx-vscode#5735] [Ideas]
I am in a situation where I am unable to start the debugger as it takes at least a minute to start up (every time), it would be really nice if I could put in a System.debug statement in my source and see the output of it when I run a test down with the test output so that if I am trying to figure out what is happening I don't need to open up dev console in the browser.

**Comment by atnmorrison** [comment]
Basically make sure it's at feature parity with force.com ide before it was dropped.

**Comment by andrewhorwood** [comment]
This would be useful. At the moment I am using the dev console to see debugs when running tests

[a comment by an automated account is not shown]

**Comment by diyer** [comment]
@matthewjcavalier , have you tried the replay debugger?
